src/social/components/Comment/index.js

```diff
@@ -260,7 +267,22 @@ const Comment = ({
   );
 
   return isReplyComment ? (
-    <ReplyContainer data-qa-anchor="reply">{renderedComment}</ReplyContainer>
+    <>
+      {isReplying && (
+        <CommentComposeBar
+          postId={parentComment?.referenceId}
+          postType={parentComment?.referenceType}
+          userToReply={commentAuthor.displayName}
+          onSubmit={(replyText, mentionees, metadata) => {
```

Comment:
This rerenders the component since it gets a new function on every render, extract it to a const ouside of return?